config/config.php

```diff
@@ -37,17 +47,20 @@
 		'linux' => [
 			'version' => $ver,
 			'versionstring' => $ver_str,
-			'web' => $url . 'Nextcloud-' . $ver . '-x86_64.AppImage',
+			'downloadurl' => $linux_url . 'Nextcloud-' . $ver . '-x86_64.AppImage',
+			'web' => 'https://nextcloud.com/install/?pk_campaign=clientupdate#install-clients',
```

Comment:
Setting `web` to point to our download overview page will force users to select and start their download manually each time.
Because of our not-working auto updater I sent them to the appropriate direct download instead (just set `web` to `downloadurl`'s value).
Do we really want to revert this back to the old behaviour?

Reply:
Yes. Now we could do it for Windows only I guess. I tested it and it definitely works when downloadurl is there.

On Mac I honestly can't check how it behaves when Sparkle is in the mix. That said, if we don't build with Sparkle it goes through the PassiveUpdateNotifier anyway. Do we build with sparkle right now? I don't think we do, am I wrong?

Same thing for AppImage in all cases it goes through PassiverUpdateNotifier. They would end up on the landing page which is IMO better than opening a browser straight to the binary.

Reply:
I remember back then complains, then we directed them directly to the binaries. But okay, it was like this PR before, so let's see if we get grilled or not ;-)

Sparkle is another work-in-progress thing, for now we don't bundle it and it just behaves the way the AppImage does.

Reply:
Well and right now we get grilled because the update isn't automated enough... go figure...